plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/common/SwitchField.tsx

```diff
@@ -0,0 +1,22 @@
+import { Switch } from "@web/sharedComponents";
+import { ComponentProps } from "react";
+
+import { FieldTitle, BorderLessFieldValue, FieldWrapper } from "../commonStyles";
```

Comment:
Can you rename the commonStyles folder to commonComponents? They use the styled components library, but they are components. `commonStyles` sounds like css only, which is a little misleading

Reply:
Done!